scripts/socketserver/stop.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+command -v shellcheck > /dev/null && shellcheck "$0"
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+pkill -f "$SCRIPT_DIR"/echo.py --port 444
```

Comment:
Can we put the search pattern into a single string, to make clear what belongs where?

```
pkill -f "$SCRIPT_DIR/echo.py --port 444"
```

should be the same. 444 is used as a common prefix for both processes, right?

Reply:
Yep, we use 4444 and 4445.